DoCoMoUnderstandingReq request
---
bool success
DoCoMoUnderstandingRes response
